src/demandlib/bdew/heat_building.py

```diff
@@ -268,11 +274,29 @@ def get_sigmoid_parameters(self, filename="shlp_sigmoid_factors.csv"):
             + "wind_impact=={0}".format(self.wind_class)
         )
 
-        a = float(sigmoid["parameter_a"])
-        b = float(sigmoid["parameter_b"])
-        c = float(sigmoid["parameter_c"])
+        wrong_number_of_parameters_message = (
+            "{} sigmoid parameters found for "
+            + f"building_class={self.building_class}, shlp_type={self.shlp_type}, "
+            + f"wind_class={self.wind_class}"
+        )
+
+        if len(sigmoid) == 0:
```

Comment:
I would rather check for `len(sigmoid) != 1` and have an error message with "`len(sigmoid)` parameters found, should be 1 for ..."
To reduce the cases for error handling and increase the information to the user (the actual number of params)